.ci/packaging.groovy

```diff
@@ -168,6 +168,7 @@ pipeline {
                   PLATFORMS = [
                     '+all',
                     'darwin/amd64',
+                    'darwin/arm64',
```

Comment:
In order to test these changes you need to use the `UI`:
- Go to https://beats-ci.elastic.co/job/Beats/job/packaging/view/change-requests/job/PR-29585/build?delay=0sec and select `macos`

Although I don't know what's the current status, since this particular stage is not executed often.

Reply:
Actually, It won't work -> https://beats-ci.elastic.co/label/macosx-10.12/ is not anymore available, so you need to change the label to use a different one, you can use https://beats-ci.elastic.co/label/macosx/, 

change https://github.com/elastic/beats/pull/29585/files#diff-5edb5ab93fc95960129eecb2ee3d8763a9c5e02b56e5b80126e0c5a52296ee8fR158 from `agent { label 'macosx-10.12' }` with `agent { label 'macosx' }`

Though I cannot warranty what's the status for those macos workers, as they were corrupted sometime ago, so, if any issues then you might need to contact the infra team as they provision and manage those macos workers.